### PR TITLE
Add log10.md

### PR DESCRIPTION
**Pull request by Sparsh47**
Description
added new term entry C++ Math Functions: log10()

#957 

Type of Change
Adding a new entry

Checklist
- [x] All writings are my own.
- [x]  My entry follows the Codecademy Docs style guide.
- [x]  My changes generate no new warnings.
- [x]  I have performed a self-review of my own writing and code.
- [x]  I have checked my entry and corrected any misspellings.
- [x]  I have made corresponding changes to the documentation if needed.
- [x]  I have confirmed my changes are not being pushed from my forked main branch.
- [x]  I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x]  Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes # in the "Description" above).

### CODE REVIEW

**Comment by Sparsh47** [comment]
I have already signed the cla agreement but it is still showing not signed yet. What to do  now?
image

**Comment by SSwiniarski** [comment]
> I have already signed the cla agreement but it is still showing not signed yet. What to do now? image

@Sparsh47, Do you have another account? It's actually complaining about @Sparsh1807. If that's also your account, signing with that may solve the issue.

File: content/cpp/concepts/math-functions/terms/log10/log10.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+The **log10()** function returns the base-_10_ logarithm of the argument.
+
+## Syntax
+
+```cpp
+ log10(x)
+```
+
+If the `x` parameter is less than or equal to zero, then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressedly returned.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/log10/log10.md:23]:
```suggestion
If the `x` parameter is less than or equal to zero, then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.
```

File: content/cpp/concepts/math-functions/terms/log10/log10.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Example
+
+The following example uses the `log10()` function to find the natural log of `5`:
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/log10/log10.md:27]:
```suggestion
The following example uses the `log10()` function to find the base-10 log of `5`:
```

File: content/cpp/concepts/math-functions/terms/log10/log10.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  double result;
+  result = log10(x);
+  cout << "The log to the base of 10 of " << x << " is " << result << "\n";
+  // Output: The log of 5 is 0.69897
+}
+```
+
+## Codebyte Example
+
+The following example is runnable and returns the natural log of `10`:
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/log10/log10.md:45]:
```suggestion
The following example is runnable and returns the base-10 log of `10`:
```

File: content/cpp/concepts/math-functions/terms/log10/log10.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **log10()** function returns the base-_10_ logarithm of the argument.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/log10/log10.md:15]:
```suggestion
The **log10()** function returns the base-10 logarithm of the argument.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Sparsh47, I did a first review and left a few comments to be addressed.

**Comment by Sparsh47** [comment]
i have successfully committed all the changes. Please review this again.

File: content/cpp/concepts/math-functions/terms/log10/log10.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **log10()** function returns the base-10 logarithm of the argument.
+
+## Syntax
+
+```cpp
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/log10/log10.md:19]:
```suggestion
```pseudo
```

File: content/cpp/concepts/math-functions/terms/log10/log10.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+ log10(x)
+```
+
+If the `x` parameter is less than or equal to zero, then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/log10/log10.md:23]:
```suggestion
If the `x` parameter is negative (less than zero), then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.
```

**Review by Dusch4593 — APPROVED** [review]
@Sparsh47 LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Thank you so much for your contribution to Codecademy Docs, @Sparsh47! 🎉 🎊

The entry should now be live: https://www.codecademy.com/resources/docs/cpp/math-functions/log10

Also, let me know if you'd like to be credited on the entry page or remain anonymous 😄

Screen Shot 2022-10-21 at 12 55 32 PM

**Comment by yangc95** [comment]
@Sparsh47 congrats on the public entry!

I had two follow-up edits for this entry:
- Move the opening bracket to a new line for both code snippets.
- Remove the term "expressly" from line 23 due to its dual meaning which can be confusing (quick or explicit), it's also unnecessary.

Let's open a new PR for these edits. We can label this as a typo and it can be merged right away. Thanks!

cc: @Dusch4593 @SSwiniarski

**Pull request merged**